{
	"id": "AVG-1828",
	"advisories": [
		{
			"id": "ASA-202104-2",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/ASA-202104-2"
				}
			]
		},
		{
			"id": "AVG-1828",
			"severity": [
				{
					"type": "vendor",
					"source": "security.archlinux.org",
					"vendor": "High"
				}
			],
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/AVG-1828"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2021-21201",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21201"
				}
			]
		},
		{
			"id": "CVE-2021-21202",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21202"
				}
			]
		},
		{
			"id": "CVE-2021-21203",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21203"
				}
			]
		},
		{
			"id": "CVE-2021-21207",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21207"
				}
			]
		},
		{
			"id": "CVE-2021-21209",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21209"
				}
			]
		},
		{
			"id": "CVE-2021-21210",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21210"
				}
			]
		},
		{
			"id": "CVE-2021-21213",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21213"
				}
			]
		},
		{
			"id": "CVE-2021-21214",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21214"
				}
			]
		},
		{
			"id": "CVE-2021-21215",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21215"
				}
			]
		},
		{
			"id": "CVE-2021-21216",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21216"
				}
			]
		},
		{
			"id": "CVE-2021-21217",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21217"
				}
			]
		},
		{
			"id": "CVE-2021-21218",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21218"
				}
			]
		},
		{
			"id": "CVE-2021-21219",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21219"
				}
			]
		},
		{
			"id": "CVE-2021-21221",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21221"
				}
			]
		},
		{
			"id": "CVE-2021-21222",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21222"
				}
			]
		},
		{
			"id": "CVE-2021-21223",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21223"
				}
			]
		},
		{
			"id": "CVE-2021-21225",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21225"
				}
			]
		},
		{
			"id": "CVE-2021-21226",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21226"
				}
			]
		},
		{
			"id": "CVE-2021-21227",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21227"
				}
			]
		},
		{
			"id": "CVE-2021-21228",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21228"
				}
			]
		},
		{
			"id": "CVE-2021-21229",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21229"
				}
			]
		},
		{
			"id": "CVE-2021-21230",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21230"
				}
			]
		},
		{
			"id": "CVE-2021-21231",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21231"
				}
			]
		},
		{
			"id": "CVE-2021-21232",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21232"
				}
			]
		},
		{
			"id": "CVE-2021-21233",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2021-21233"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "arch",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "vivaldi"
						},
						"affected": {
							"type": "pacman",
							"range": [
								{
									"lt": "3.8.2259.37-1"
								}
							],
							"fixed": [
								"3.8.2259.37-1"
							]
						}
					}
				]
			}
		}
	]
}
